src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -78,8 +78,12 @@ studio.configuration.global.ui.resource.override.path: crafter/studio/ui
 studio.configuration.global.systemSite: studio_root
 # Root location where configuration for all sites is stored inside repository.
 studio.configuration.site.configBasePath: /config/studio
-# Root location where configuration for all sites is stored inside repository.
+# Root location pattern where configuration for all sites is stored inside repository.
+studio.configuration.site.configBasePathPattern: /config/{module}
+# Root location where configuration for all sites is stored inside repository for specified environment.
 studio.configuration.site.multiEnvironment.configBasePath: /config/studio/env/{environment}
```

Comment:
Why not just have one general pattern for all modules. We should follow this pattern across all modules, no?
`/config/{module}/env/{env}/`

Reply:
Small steps. When we fully refactor configuration service, we will have a general pattern